File: c/iniciante/BEE1010.c
#include <stdio.h>

int main() {
  int codigo_peca1, codigo_peca2, numero_de_pecas1, numero_de_pecas2;
  double valor_peca1, valor_peca2;

  scanf("%d %d %lf", &codigo_peca1, &numero_de_pecas1, &valor_peca1);
  scanf("%d %d %lf", &codigo_peca2, &numero_de_pecas2, &valor_peca2);

  double total = (numero_de_pecas1 * valor_peca1) + (numero_de_pecas2 * valor_peca2);

  printf("VALOR A PAGAR: R$ %.2lf\n", total);

  return 0;
}

File: go/iniciante/BEE1010.go
package main

import ("fmt")

func main() {
	var codigoPeca1 int
	var numeroPecas1 int
	var valorUnitario1 float64
	fmt.Scanf("%d %d %f", &codigoPeca1, &numeroPecas1, &valorUnitario1)

	var codigoPeca2 int
	var numeroPecas2 int
	var valorUnitario2 float64
	fmt.Scanf("%d %d %f", &codigoPeca2, &numeroPecas2, &valorUnitario2)

	var total float64 = (float64(numeroPecas1) * valorUnitario1) + (float64(numeroPecas2) * valorUnitario2)

	fmt.Printf("VALOR A PAGAR: R$ %.2f\n", total)
}